tesseract/tesseract_collision/src/bullet/bullet_discrete_bvh_manager.cpp

```diff
@@ -156,6 +156,7 @@ bool BulletDiscreteBVHManager::enableCollisionObject(const std::string& name)
   if (it != link2cow_.end())
   {
     it->second->m_enabled = true;
+    broadphase_->getOverlappingPairCache()->cleanProxyFromPairs(it->second->getBroadphaseHandle(), dispatcher_.get());
```

Comment:
What does this do and why is it necessary? Perhaps a comment

Reply:
Done